Pipfile

```diff
@@ -9,6 +9,7 @@ verify_ssl = true
 id3c = {git = "https://github.com/seattleflu/id3c"}
 datasette = ">=0.41"
 sqlite-utils = "*"
+psycopg2 = "*"
```

Comment:
I don't think this should be necessary, since the only thing using psycopg2 here is ID3C, and we already declare a dep on that. IIRC, psycopg2 has no external deps, so judging on the many unrelated additions to the Pipfile.lock in this commit, I think maybe the Pipfile.lock as committed was accidentally out of date or something. (Or it was "up to date", but then pipenv/pip got better at recording transitive dependencies or something?)

Reply:
`pipenv run make -b` did not work for me when psycopg2 was not installed as a dependency via pipenv. 
I'm not sure about the bulk of the changes in Pipfile.lock. Perhaps the committed one was out of date.

Reply:
Ah. I think Pipfile.lock was in fact out of date. I did not need to add psycopg2 as a dependency after all. After running `pipenv update`, the `make` file runs as expected. I dropped the previous Pipfile.lock commit in favor of a new one.